ProjectSailor/Source/ProjectSailor/InteractionComponent.h

```diff
@@ -0,0 +1,62 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "ObjectInteraction.h"
+#include "Components/ActorComponent.h"
+// #include "InteractionInterface.h"
+// #include "InteractWidget.h"
+// #include "MaterialSphere.h"
+#include "InteractionComponent.generated.h"
+
+
+UCLASS( ClassGroup=(Custom), meta=(BlueprintSpawnableComponent) )
+class PROJECTSAILOR_API UInteractionComponent : public UActorComponent
+{
+    GENERATED_BODY()
+
+public:	
+    // Sets default values for this component's properties
+    UInteractionComponent();
+
+protected:
+    // Called when the game starts
+    virtual void BeginPlay() override;
+
+    // UPROPERTY(EditDefaultsOnly, Category= "Interaction") TSubclassOf<UInteractWidget> widgetTemplate;
+    // UPROPERTY()UInteractWidget *widget = nullptr;
```

Comment:
What's the purpose of these properties ? If we are not using them, lets delete them